recipes/swepy/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "swepy" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91ed36a907399ce7a29d763cf74077ce14d55e8bcdf52b6f1e8b5723f48d322f
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
This should be `{{ PYTHON }} -m pip install --no-deps --ignore-installed .`

Reply:
This has been fixed as well! Thanks for the assistance